src/NetherPortalScanner.h

```diff
@@ -39,8 +39,8 @@ class cNetherPortalScanner : public cChunkStay
 	static const int SearchSolidBaseWidth = 3;
 
 	/** Where to place the player out from the face and across the face */
-	const double OutOffset = 2;  // NOLINT
-	const double AcrossOffset = 0.5;  // NOLINT
+	const double OutOffset = 2;
+	const double AcrossOffset = 0.5;
```

Comment:
Why is this not `static`? The last build just failed, because `readability-identifier-naming.ClassConstantPrefix` only works on `static` constant fields.

Reply:
`/home/bond/Documents/GitHub/cuberite/src/Entities/../NetherPortalScanner.h:42:22: error: in-class initializer for static data member of type 'const double' requires 'constexpr' specifier [-Wstatic-float-init]
        static const double OutOffset = 2;`
`constexpr` is uses nowhere else in the code so I just left it and chenged the check.

Reply:
This only means, that you should put the definition outside the class:
```cpp
class X
{
    static const int z;
}
const int X::z = 42; // If you have a Cpp file, put this line into it.

Reply:
@bibo38 How does that work when the member is private?

Reply:
> How does that work when the member is private?

What do you mean? It's like private member functions, which you define outside of the class.

Btw:
> If a static data member **of integral or enumeration type** is declared const (and not volatile), it can be initialized with an initializer in which every expression is a constant expression, right inside the class definition

(http://en.cppreference.com/w/cpp/language/static)
:rofl:

Reply:
Yeah, it wasn't working because I made a typo :/, thanks.